hack/local-up-kubeedge.sh

```diff
@@ -90,6 +93,7 @@ function start_cloudcore {
   CLOUD_CONFIGFILE=${KUBEEDGE_ROOT}/_output/local/bin/cloudcore.yaml
   CLOUD_BIN=${KUBEEDGE_ROOT}/_output/local/bin/cloudcore
   ${CLOUD_BIN} --minconfig >  ${CLOUD_CONFIGFILE}
+  sed -i '/modules:/a\  cloudStream:\n    enable: true\n    streamPort: 10003\n    tlsStreamCAFile: /etc/kubeedge/ca/streamCA.crt\n    tlsStreamCertFile: /etc/kubeedge/certs/stream.crt\n    tlsStreamPrivateKeyFile: /etc/kubeedge/certs/stream.key\n    tlsTunnelCAFile: /etc/kubeedge/ca/rootCA.crt\n    tlsTunnelCertFile: /etc/kubeedge/certs/server.crt\n    tlsTunnelPrivateKeyFile: /etc/kubeedge/certs/server.key\n    tunnelPort: 10004' ${CLOUD_CONFIGFILE}
```

Comment:
Can be simpified, only replace what we need.

Reply:
@daixiang0 `cloudcore --minconfig` does not print the `cloudStream` section

Reply:
I mean that you write too mush in `sed` replace.

Reply:
I know that. But there was only `cloudHub` moudle in `cloudcore --minconfig`, I have to append all these lines of `cloudStream` moudle configuration.

Reply:
I see.